mender-deb-package

```diff
@@ -61,9 +61,8 @@ prepare_recipe() {
       cp support/mender-client.service debian/mender-client.service ||
       cp support/mender.service debian/mender-client.service
     fi
-  else
-    # Other packages here
-    true
+  elif [ ${DEB_PACKAGE} = "mender-shell" ]; then
```

Comment:
Nitpick:
Quote both maybe?

Reply:
Okey, I'll add them.

As the script checks the arguments, there is no way that DEB_PACKAGE can be empty.